src/style/App.scss

```diff
@@ -59,6 +60,10 @@ body {
   text-align: center;
 }
 
+.initial-page-container__description-text-container {
+  text-align: -webkit-center;
```

Comment:
I guess you couldn't align it with _text-align: 'center'_, I would opt for playing around with css props to achieve the same behavior instead of going for prefixes properties. Anyway, if you want to leave it as it is, Maybe it would be fine define -_moz-center_ too, so at least you extend your browser market share compatibility. Take a look at this http://help.dottoro.com/lcnebmiu.php